src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -18,6 +18,24 @@ function isPreview() {
 }
 
 $(document).ready(function() {
+
+	/**
+	 * Handles Keyup and Keydown on Text question to display response length
```

Comment:
Hmm, the indentation is still entirely off here? Does it look correct when you are editing it in your text editor/ide?